### DISCUSSION TITLE
Is style recovery still working as intended?

### DISCUSSION THREAD

**Discussion by narcolepticinsomniac** [openstyles/stylus#1482] [General]
@tophf IIRC, you implemented some sorta last-ditch style recovery a while back, in case of a crash, or other similar type of failure. In the last few weeks, I've seen a couple separate complaints on both AMO, and CWS, claiming this feature is currently FUBAR.

**Comment by tophf** [comment]
Well, FUBAR is a vague "term".

**Reply by narcolepticinsomniac** [reply]
No doubt. I believe the CWS complaint was pretty vague, just something about it not working. The one on AMO today claims it borked the entire extension.
